src/core.rs

```diff
@@ -9,12 +9,20 @@ use smallvec::SmallVec;
 use crate::errors::Error;
 use crate::errors::SBSError;
 
-use crate::inversion_tree::InversionTree;
 use crate::matrix::Matrix;
 
+use lru::LruCache;
+
+#[cfg(feature = "std")]
+use parking_lot::Mutex;
+#[cfg(not(feature = "std"))]
+use spin::Mutex;
+
 use super::Field;
 use super::ReconstructShard;
 
+const DATA_DECODE_MATRIX_CACHE_CAPACITY: usize = 254;
```

Comment:
This is an odd number. Can you elaborate on the rationale behind this number, maybe leaving a comment in the code too?

Reply:
I just preserved the current cache capacity from the existing code, not to introduce any change of behavior or performance due to cache size change:
https://github.com/rust-rse/reed-solomon-erasure/blob/eb1f66f47/src/inversion_tree.rs#L15